tests/unit/detection/test_detection_model.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import pytest
+from torch import tensor
+from torchvision.models.detection.faster_rcnn import FasterRCNN
+
+from utils_cv.detection.dataset import DetectionDataset
+from utils_cv.detection.bbox import DetectionBbox
+from utils_cv.detection.model import (
+    get_pretrained_fasterrcnn,
+    DetectionLearner,
+    _get_det_bboxes,
+    _apply_threshold,
+    _calculate_ap,
+)
+
+
+@pytest.fixture(scope="session")
+def od_detection_dataset(tiny_od_data_path):
+    """ returns a basic detection dataset. """
+    return DetectionDataset(tiny_od_data_path)
+
+
+@pytest.fixture(scope="session")
+def od_detection_learner(od_detection_dataset):
+    """ returns a basic detection dataset. """
+    learner = DetectionLearner(od_detection_dataset)
```

Comment:
to speed things up for unit tests, could here reduce image resolution to e.g. 100 pixels (smaller causes errors) and also set the number of proposals to consider to something small.